<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnapmaticEditor</class>
 <widget class="QDialog" name="SnapmaticEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Snapmatic Properties</string>
  </property>
  <layout class="QVBoxLayout" name="vlEditor">
   <item>
    <widget class="QGroupBox" name="gbMode">
     <property name="title">
      <string>Snapmatic Type</string>
     </property>
     <layout class="QGridLayout" name="gdType">
      <item row="2" column="1">
       <widget class="QRadioButton" name="rbEditor">
        <property name="text">
         <string>Editor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rbSelfie">
        <property name="text">
         <string>Selfie</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbCustom">
        <property name="text">
         <string>Regular</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="rbMugshot">
        <property name="text">
         <string>Mugshot</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbProperties">
     <property name="title">
      <string>Snapmatic Properties</string>
     </property>
     <layout class="QGridLayout" name="gdProperties">
      <item row="5" column="1">
       <widget class="QCheckBox" name="cbEditor">
        <property name="text">
         <string>Editor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="cbMeme">
        <property name="text">
         <string>Meme</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="cbDirector">
        <property name="text">
         <string>Director</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="cbSelfie">
        <property name="text">
         <string>Selfie</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QCheckBox" name="cbMugshot">
        <property name="text">
         <string>Mugshot</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbExtras">
     <property name="title">
      <string>Extras</string>
     </property>
     <layout class="QVBoxLayout" name="vlExtras">
      <item>
       <widget class="QCheckBox" name="cbQualify">
        <property name="text">
         <string>Qualify as Avatar automatically at apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labQualify">
        <property name="text">
         <string>Qualify as Avatar allows you to use this Snapmatic as a Social Club profile picture</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="vsEditor">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlButtons">
     <item>
      <spacer name="hsButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdApply">
       <property name="text">
        <string>&amp;Apply</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-apply"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-cancel"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
